<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-8EA77APH" ignored="2" name="Default test(failed)" tests="4" failures="1" timestamp="28 Oct 2021 09:57:15 GMT" time="0.062" errors="0">
  <testcase name="tc2" time="0.016" classname="org.dataprovider.TestCase">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.0" classname="org.dataprovider.TestCase">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.0" classname="org.dataprovider.TestCase">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.0" classname="org.dataprovider.TestCase">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.dataprovider.TestCase.tc2(TestCase.java:14)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc2 -->
  <testcase name="tc3" time="0.0" classname="org.dataprovider.TestCase">
    <ignored/>
  </testcase> <!-- tc3 -->
  <testcase name="tc1" time="0.0" classname="org.dataprovider.TestCase">
    <ignored/>
  </testcase> <!-- tc1 -->
</testsuite> <!-- Default test(failed) -->
